### ISSUE TITLE
Stopped working due to changes in Google's code

### ISSUE THREAD

**Issue by SigmaTheDJ** [danieljbingham/google-image-sizer#14]
Unfortunately this stopped working a few days ago. Any chance of an update please?

**Comment by kneckergit** [comment]
Just confirming the issue here. Is also being confirmed by dumb people downvoting the addon on addons.firefox.com. tz.

**Comment by danieljbingham** [comment]
Yeah unfortunately looks Google have made quite a significant change to how they structure their image search pages - will probably need quite a rewrite, looking into it

**Comment by danieljbingham** [comment]
Got a fix that is a bit flakey but works, will release this to the store for the time being, while working to make it a bit more reliable. Also need to fix the size display on each individual image

Currently under review in the google store

**Comment by SigmaTheDJ** [comment]
Google have changed things up again. This is what my tools menu looks like now: -

https://i.imgur.com/lSJkVFf.jpg

It's the same on both Chrome and Firefox.

**Comment by perigozin** [comment]
Any fix or is this project dead?
